roles/openshift_health_checker/test/search_journalctl_test.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+import search_journalctl
+
+
+def canned_search_journalctl(get_log_input=None):
+    """Create a search_journalctl object with canned get_log_input method"""
+    module = search_journalctl
+    if get_log_input:
+        module.get_log_input = get_log_input
+    return module
+
+
+DEFAULT_TIMESTAMP = 1496341364
+
+
+def get_timestamp(modifier=0):
+    return DEFAULT_TIMESTAMP + modifier
+
+
+def get_timestamp_microseconds(modifier=0):
+    return get_timestamp(modifier) * 1000000
+
+
+def create_test_log_object(stamp, msg):
+    return '{{"__REALTIME_TIMESTAMP": "{}", "MESSAGE": "{}"}}'.format(stamp, msg)
+
+
+@pytest.mark.parametrize('matchers,log_input,expected_matches,expected_errors', [
+    (
+        [
+            {
+                "start_regexp": r"Sample Logs Beginning",
+                "regexp": r"test log message",
+                "unit": "test",
+            },
+        ],
+        [
+            create_test_log_object(get_timestamp_microseconds(), "test log message"),
+            create_test_log_object(get_timestamp_microseconds(), "Sample Logs Beginning"),
+        ],
+        ["test log message"],
+        [],
+    ),
+    # test with invalid json in log input
```

Comment:
Make the comment part of the test data and the test case ID, so that it becomes part of the output on failure.